src/runtime-rs/crates/runtimes/linux_container/Cargo.toml

```diff
@@ -10,3 +10,4 @@ async-trait = "0.1.48"
 tokio = { version = "1.8.0" }
 
 common = { path = "../common" }
+kata-types = { path = "../../../../libs/kata-types" }
```

Comment:
It would be create if cargo provided a `$top_level_dir` feature imho ;)

Reply:
This is a nice change, unfortunately not supported yet.